docs/yugabyte-paths.md

```diff
@@ -0,0 +1,20 @@
+<h2>Cluster Install Directories</h2>
```

Comment:
This is very useful! But this information would be ideally added to the following page: https://docs.yugabyte.com/latest/admin/yb-ctl/
The above page corresponds to the following file: `content/latest/admin/yb-ctl.md`

Do you mind adding this info there? Otherwise please let us know and we can do that.